packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -42,6 +42,7 @@
 	}
 }
 
+.edit-post-sidebar,
```

Comment:
Why do we need this class here? It's an edit-post class and ideally, it shouldn't be used in `block-editor`

Reply:
Yep, I knew this was going to come up. I've been thinking about what might be better. Right now the toolbar CSS — the paddings, the dimensions, the focus style — is a bit complex due to the contents of the toolbar having lots of un-classed fragments, and because not all toolbar segments are `component-toolbars`. 

But all that is the reason why the CSS that styles the formatting buttons are scoped to toolbars labelled `block-editor-block-toolbar` or `block-editor-format-toolbar`, as opposed to being attached to a more generic `components-toolbar`. 

So either we can refactor the toolbar component to have a lighter more predictable DOM and optimize the styles, or we can duplicate the whole set of rules in this file just for the sidebar.

Reply:
> not all toolbar segments are component-toolbars.

I think at some point, I updated most of them and wrapped them with `Toolbar` component. We might be missing some though.

Reply:
Given this is sort of an ugly regression, in absence of a full on refactor of the toolbars to be more componentized, I can do a smaller hack just for the buttons in the sidebar, and add a comment to remove it once refactored? To me that would be preferable to duplicating this complicated code block. What do you think?

Reply:
Yes, I'm fine with the hack and 👍 for a comment. 
I'm just thinking that ultimately we could try scoping these styles for any `Toolbar` component and see if it addresses the issues and what regressions this could cause.